sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/PrivateService.java

```diff
@@ -107,6 +107,26 @@ public static <T> Optional<T> getCompasPrivate(TPrivate tPrivate, Class<T> compa
         return Optional.of(compasPrivates.get(0));
     }
 
+    /**
+     * Converts Private of type <em>compasClass</em> given as parameter
+     * @param baseElement TBaseElement contenting Privates
+     * @param compasClass type in which Privates should be given
+     * @return optional of formatted Private
+     * @param <T> Inference parameter stands for wanted type of Private
+     * @throws ScdException throws when inconsistency between types
+     */
+    public static <T> Optional<T> getCompasPrivate(TBaseElement baseElement, Class<T> compasClass) throws ScdException {
```

Comment:
Rename method, it is not just a getter, use javadoc to find better one ?

Reply:
This should be a getter on `TBaseElement` class. Unfortunately, this class is generated by xjc (it is a JAXB class), so I cannot create the getter on `TBaseElement` . That's why I created this static class `PrivateService`.  
We can rename it as `extractCompasPrivates`, what do you think ?